contracts/token/RarePizzasBox.sol

```diff
@@ -51,7 +51,7 @@ contract RarePizzasBox is
 
     // IRarePizzasBox
     function getPrice() public view override returns (uint256) {
-        return super.curve(super.totalSupply());
+        return super.curve(super.totalSupply() + 1);
```

Comment:
Would prefer not to go past totalSupply too much

Reply:
similar to the above i think this is fine for now.  this logic is going to change as part of #18